test/integration/discovery_test.go

```diff
@@ -45,6 +45,11 @@ func TestDiscoveryGroupVersions(t *testing.T) {
 	expectedGroupVersions := sets.NewString()
 	for group, versions := range configapi.KubeAPIGroupsToAllowedVersions {
 		for _, version := range versions {
+			// TODO: The policy is enabled by kube but does not have client registered
```

Comment:
This is the new policy api group added upstream that includes the podDisruptionBudget type. Doesn't have to do with what we mean by "policy" in OpenShift but either way it needs its own issue:)